tools/releaseBuild/azureDevOps/templates/compliance.yml

```diff
@@ -8,11 +8,15 @@ jobs:
     value : false
   - name: NugetSecurityAnalysisWarningLevel
     value: none
+  - group: PS-PS-APIScan
```

Comment:
Can you add a comment on what variables this brings in?

Reply:
Added.